src/charm.py

```diff
@@ -127,24 +164,32 @@ def _on_gunicorn_pebble_ready(self, event: ops.framework.EventBase) -> None:
 
     def _configure_workload(self, event: ops.charm.EventBase) -> None:
         """Configure the workload container."""
-        pebble_config = self._get_pebble_config(event)
+        pebble_config = self._get_gunicorn_pebble_config(event)
+        statsd_pebble_config = self._get_statsd_pebble_config(event)
         if not pebble_config:
```

Comment:
If we change `pebble_config` above to `gunicorn_pebble_config` this will obviously need changing as well.